runtime/helpers.go

```diff
@@ -22,10 +22,15 @@ import (
 	"github.com/firecracker-microvm/firecracker-go-sdk"
 	models "github.com/firecracker-microvm/firecracker-go-sdk/client/models"
 
+	"github.com/firecracker-microvm/firecracker-containerd/config"
 	"github.com/firecracker-microvm/firecracker-containerd/proto"
 )
 
-func machineConfigurationFromProto(cfg *Config, req *proto.FirecrackerMachineConfiguration) models.MachineConfiguration {
+const (
+	defaultMemSizeMb = 128
```

Comment:
This is minor but why move this?

Reply:
Basically due to the `Config` needing to be relocated to the `config` package, this `defaultMemSizeMb` was also in the `config.go` file. And while keeping it there makes sense at first, I realized it wasn't used in the `config.go` file at all and only here. So basically, either we expose `defaultMemSizeMb` or we relocate it. And I thought relocating was better than exposing.

Reply:
It's unrelated to this PR, but it seems weird that defaultMemSizeMb isn't used in tehs ame place as the other defaults?

Is the use of `defualtMemSizeMb` in `machineCOnfigurationFromProto` only because it's not used in `LoadConfig` with the other defaults? Would it make sense to move it's use there rather than moving the definition?

Reply:
The `Config` does not have a `MemSizeMib` field. So we'd be introducing another field in the configuration, which I do not like either if it isn't needed. Currently you can only overwrite the default value by `req.MemSizeMib`

Reply:
Do you happen to know off-hand why mem size was treated differently than other config values? It seems like it *should* be part of the config and would be simpler if it was. Maybe there is a reason I'm not seeing/aware of?

Reply:
I think it's due to complications of priority. Basically, getting the memory size would need to check if `req.MemSizeMib` is zero, if it is use the `cfg.MemSizeMib`. We probably want to take all of the config values out that can be used in `CreateVM`. Meaning all the machine config stuff would now be apart of `CreateVM`. Then the config would only have values that wouldn't change, like `ShimBaseDir` or the runc bin path. Stuff like that

Reply:
Agree on out of scope of this PR, was just curious about it! Resolving.